test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -744,10 +749,221 @@ def __are_requirements_matching(param_dic=None, test=None):
+        # mom hardware report
+        for hostname in self.param_dict['moms']:
+            hr = SystemInfo()
+            hr.get_system_info(hostname)
+            # monitors ram
+            avail_ram = getattr(hr, 'system_ram', None)
+            total_ram = getattr(hr, 'system_total_ram', None)
+            if (avail_ram or total_ram) is None:
+                _msg = "hostname " + hostname
+                self.logger.error(_msg + "unable to get ram info")
+                self.hardware_report_timer.cancel()
+                raise PbsSystemError(msg=_msg+"unable to get ram info")
+            else:
+                used_ram_percent = ((total_ram - avail_ram)/total_ram)*100
+                if 70 < used_ram_percent < 95:
+                    _msg = "hostname- " + hostname + ": used ram is "
+                    self.logger.warning(_msg + str(used_ram_percent) + "%")
+                elif used_ram_percent > 95:
+                    _msg = "hostname- " + hostname + ": used ram is "
+                    self.logger.error(_msg + str(used_ram_percent) + "%")
+                    self.hardware_report_timer.cancel()
+                    raise PbsSystemError(msg=_msg+str(used_ram_percent)+"%")
+            # monitors disk
+            used_disk_percent = getattr(hr, 'system_disk_used_percent', None)
+            if used_disk_percent is None:
+                _msg = "hostname " + hostname
+                self.logger.error(_msg + ": unable to get disk info")
+                self.hardware_report_timer.cancel()
+                raise PbsSystemError(msg=_msg+"unable to get disk info")
+            elif 70 < used_disk_percent < 95:
+                _msg = "hostname- " + hostname + ": used disk is "
+                self.logger.warning(_msg + str(used_disk_percent) + "%")
+            elif used_disk_percent > 95:
+                _msg = "hostname- " + hostname + ": used disk is "
+                self.logger.error(_msg + str(used_disk_percent) + "%")
+                self.hardware_report_timer.cancel()
+                raise PbsSystemError(msg=_msg+str(used_disk_percent)+"%")
+            # checks core files in mom_priv
+            pbs_conf = du.parse_pbs_config(hostname)
+            mom_priv_path = os.path.join(pbs_conf["PBS_HOME"],
+                                         "mom_priv")
+            mom_priv_files = du.listdir(hostname,
+                                        mom_priv_path, sudo=True,
+                                        fullpath=False)
+            for filename in mom_priv_files:
+                if filename.startswith("core"):
+                    _msg = "hostname- " + hostname + ": core file exist at "
+                    self.logger.warning(_msg + mom_priv_path)
+                    self.hardware_report_timer.cancel()
+                    raise PbsSystemError(msg=_msg + mom_priv_path)
+            # checks core files in server_priv
+            server_priv_path = os.path.join(pbs_conf["PBS_HOME"],
```

Comment:
This will fail if you have only an execution installation... as server_priv won't exist in execution-only install